cmake/dts.cmake

```diff
@@ -209,6 +210,11 @@ if(SUPPORTS_DTS)
     message(FATAL_ERROR "new extractor failed with return code: ${ret}")
   endif()
 
+  # A file that used to be generated by 'dtc'. zephyr.dts is the new
+  # equivalent.
+  file(WRITE ${PROJECT_BINARY_DIR}/${BOARD}.dts_compiled
+       "See zephyr.dts for the final merged devicetree.")
+
```

Comment:
Let's add a comment about removing this after the next Zephyr release.

We don't want this noise here forever.

Reply:
Added "will be removed in Zephyr 2.3."